druid/src/widget/textbox.rs

```diff
@@ -363,7 +363,6 @@ impl<T: TextStorage + EditableText> TextBox<T> {
             key if HotKey::new(SysMods::Cmd, "y").matches(key) && cfg!(windows) => {
                 Some(sys::REDO.to(our_id))
             }
-            key if HotKey::new(SysMods::Cmd, "a").matches(key) => Some(sys::SELECT_ALL.to(our_id)),
```

Comment:
hm, this *should* be working, so if it isn't it's probably that we just aren't handling this command? We'll need this anyway, since select-all can arrive from a menu (and hence a `Command`), and then it shouldn't be necessary to touch `simulate_input`.

Reply:
yeah, we are not handling `SELECT_ALL`.